src/test/resources/sample-userinfo.json

```diff
@@ -0,0 +1,27 @@
+{
+  "ok": true,
+  "user": {
+    "username": "sampleuser",
+    "name": "sample.user@snyk.io",
+    "email": "sample.user@snyk.io",
+    "created": "2018-07-16T13:32:49.082Z",
+    "avatar": "https://s.gravatar.com/avatar/XXX/gravatar_l.png",
+    "id": "549eb0a3-0f94-41a6-a6d7-0a2505850524",
+    "apiKeys": [
+      {
+        "key": "c3dda6a0-0aba-446f-b347-c2d4e772080a",
+        "lastUsed": "2018-07-16T13:32:49.000Z"
+      }
+    ],
+    "orgs": [
+      {
+        "name": "sampleuser",
+        "created": "2018-07-16T13:32:49.228Z",
+        "id": "7d2087a6-443c-41ef-a3e5-bb0316650b4",
+        "orgMembership": {
+          "role": "DEFAULT"
+        }
+      }
+    ]
+  }
+}
```

Comment:
Make sure this is dummy data, fake uuids and such.

Reply:
All freshly generated UUIDs from the command line :)